data/typelibrary/utils-1.0.0/typelib/splitting/LWORD_TO_BOOLS.fct

```diff
@@ -0,0 +1,295 @@
+			<VarDeclaration Name="OUT_51" Type="BOOL" Comment="BIT 51"/>
+			<VarDeclaration Name="OUT_52" Type="BOOL" Comment="BIT 52"/>
+			<VarDeclaration Name="OUT_53" Type="BOOL" Comment="BIT 53"/>
+			<VarDeclaration Name="OUT_54" Type="BOOL" Comment="BIT 54"/>
+			<VarDeclaration Name="OUT_55" Type="BOOL" Comment="BIT 55"/>
+			<VarDeclaration Name="OUT_56" Type="BOOL" Comment="BIT 56"/>
+			<VarDeclaration Name="OUT_57" Type="BOOL" Comment="BIT 57"/>
+			<VarDeclaration Name="OUT_58" Type="BOOL" Comment="BIT 58"/>
+			<VarDeclaration Name="OUT_59" Type="BOOL" Comment="BIT 59"/>
+			<VarDeclaration Name="OUT_60" Type="BOOL" Comment="BIT 60"/>
+			<VarDeclaration Name="OUT_61" Type="BOOL" Comment="BIT 61"/>
+			<VarDeclaration Name="OUT_62" Type="BOOL" Comment="BIT 62"/>
+			<VarDeclaration Name="OUT_63" Type="BOOL" Comment="BIT 63"/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION LWORD_TO_BOOLS
+
+VAR_INPUT
+	IN : LWORD;    // LWORD Input
+END_VAR
+
+VAR_OUTPUT
+	OUT_00 : BOOL;    // BIT 00
+	OUT_01 : BOOL;    // BIT 01
+	OUT_02 : BOOL;    // BIT 02
+	OUT_03 : BOOL;    // BIT 03
+	OUT_04 : BOOL;    // BIT 04
+	OUT_05 : BOOL;    // BIT 05
+	OUT_06 : BOOL;    // BIT 06
+	OUT_07 : BOOL;    // BIT 07
+	OUT_08 : BOOL;    // BIT 08
+	OUT_09 : BOOL;    // BIT 09
+	OUT_10 : BOOL;    // BIT 10
+	OUT_11 : BOOL;    // BIT 11
+	OUT_12 : BOOL;    // BIT 12
+	OUT_13 : BOOL;    // BIT 13
+	OUT_14 : BOOL;    // BIT 14
+	OUT_15 : BOOL;    // BIT 15
+	OUT_16 : BOOL;    // BIT 16
+	OUT_17 : BOOL;    // BIT 17
+	OUT_18 : BOOL;    // BIT 18
+	OUT_19 : BOOL;    // BIT 19
+	OUT_20 : BOOL;    // BIT 20
+	OUT_21 : BOOL;    // BIT 21
+	OUT_22 : BOOL;    // BIT 22
+	OUT_23 : BOOL;    // BIT 23
+	OUT_24 : BOOL;    // BIT 24
+	OUT_25 : BOOL;    // BIT 25
+	OUT_26 : BOOL;    // BIT 26
+	OUT_27 : BOOL;    // BIT 27
+	OUT_28 : BOOL;    // BIT 28
+	OUT_29 : BOOL;    // BIT 29
+	OUT_30 : BOOL;    // BIT 30
+	OUT_31 : BOOL;    // BIT 31
+	OUT_32 : BOOL;    // BIT 32
+	OUT_33 : BOOL;    // BIT 33
+	OUT_34 : BOOL;    // BIT 34
+	OUT_35 : BOOL;    // BIT 35
+	OUT_36 : BOOL;    // BIT 36
+	OUT_37 : BOOL;    // BIT 37
+	OUT_38 : BOOL;    // BIT 38
+	OUT_39 : BOOL;    // BIT 39
+	OUT_40 : BOOL;    // BIT 40
+	OUT_41 : BOOL;    // BIT 41
+	OUT_42 : BOOL;    // BIT 42
+	OUT_43 : BOOL;    // BIT 43
+	OUT_44 : BOOL;    // BIT 44
+	OUT_45 : BOOL;    // BIT 45
+	OUT_46 : BOOL;    // BIT 46
+	OUT_47 : BOOL;    // BIT 47
+	OUT_48 : BOOL;    // BIT 48
+	OUT_49 : BOOL;    // BIT 49
+	OUT_50 : BOOL;    // BIT 50
+	OUT_51 : BOOL;    // BIT 51
+	OUT_52 : BOOL;    // BIT 52
+	OUT_53 : BOOL;    // BIT 53
+	OUT_54 : BOOL;    // BIT 54
+	OUT_55 : BOOL;    // BIT 55
+	OUT_56 : BOOL;    // BIT 56
+	OUT_57 : BOOL;    // BIT 57
+	OUT_58 : BOOL;    // BIT 58
+	OUT_59 : BOOL;    // BIT 59
+	OUT_60 : BOOL;    // BIT 60
+	OUT_61 : BOOL;    // BIT 61
+	OUT_62 : BOOL;    // BIT 62
+	OUT_63 : BOOL;    // BIT 63
+END_VAR
+
+(* this Function extracts the 64 BOOLS from a lword *)
```

Comment:
this should go at the top of the function, and it should read "an LWORD" instead of "a lword". as when speaking the word the first sound, sounds like an E

Reply:
agree.